larpandora/LArPandoraEventBuilding/LArPandoraShower/Tools/ShowerTrackTrajPointDirection_tool.cc

```diff
@@ -111,7 +111,7 @@ namespace ShowerRecoTools {
       Direction_vec = InitialTrack.DirectionAtPoint(fTrajPoint);
     }
 
-    TVector3 Direction = {Direction_vec.X(), Direction_vec.Y(), Direction_vec.Z()};
+    geo::Vector_t Direction = {Direction_vec.X(), Direction_vec.Y(), Direction_vec.Z()};
```

Comment:
Good catch.  As `Direction_vec` is already of  type `geo::Vector_t`, I  suggest making the replacement above:

```diff
- geo::Vector_t Direction_vec; // Line 87
+ geo::Vector_t Direction;
```

and then adjust the `Direction_vec = ...` lines to `Direction = ...`,  and simply remove line 114.

Reply:
good spot, will do!